website/source/guides/external/lxc.html.md

```diff
@@ -20,57 +20,17 @@ documentation.
 - Nomad [LXC][lxc-docs] external driver documentation
 - Nomad LXC external driver [repo][lxc-driver-repo]
 
```

Comment:
The intro section could be beefed up a bit more. How about adding a sentence like `"This guide walks through steps involved in configuring a Nomad client agent to be able to run lxc jobs`